docker/logs/nginx/.gitignore

```diff
@@ -0,0 +1,2 @@
+!symfony_access.log
```

Comment:
these files should be ignored

Reply:
i have deleted docker/logs/nginx/.gitignore and put files to ignore in the main gitignore file